reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md

```diff
@@ -32,76 +32,122 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
 So in order for remoting over WSMan to work properly on Windows versions prior to Windows 10,
 you need to make sure the [prerequisites](#prerequisites) are met.
 
+## Deploying on Windows IoT
+
+Windows IoT already comes with Windows PowerShell which we will use to deploy PowerShell Core 6.
+
+- Create `PSSession` to target device
+
+  ```powershell
+  $s = New-PSSession -ComputerName <deviceIp> -Credential Administrator
+  ```
+
+- Copy the zip package to the device
+
+  ```powershell
+  # change the destination to however you had partitioned it with sufficient
+  # space for the zip and the unzipped contents
+  # the path should be local to the device
+  Copy-Item .\PowerShell-6.0.2-win-arm32.zip -Destination u:\users\administrator\Downloads -ToSession $s
+  ```
+
+- Connect to the device and expand the archive
```

Comment:
Can we make this a numbered list? Feels weird as bullets. Even H3s make a little more sense to me...

Unless this is a pattern I'm not aware of, in which case, ignore me.

Reply:
This is now a numbered list.